pom.xml

```
@@ -1,33 +1,33 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.2</version>
-        <relativePath />
+        <version>4.46</version>
+        <relativePath/>
     </parent>
 
     <groupId>igalg.jenkins.plugins</groupId>
     <artifactId>multibranch-build-strategy-extension</artifactId>
-    <version>1.0.11-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
```

```suggestion
    <version>${changelist}</version>
```

---

I don't know, maybe @igalg want to have fully automated version but prefer to keep at least the revision
So I've kept it like explain in the [Manually controlled prefix (optional)](https://www.jenkins.io/doc/developer/publishing/releasing-cd/#update-maven-pom-and-config:~:text=Manually%20controlled%20prefix%20(optional)) paragraph

---

this is not what is there.
To match that you need:

```suggestion
    <version>${revision}.${changelist}</version>
```